### ISSUE TITLE
Throw exception when use multiple safe navigation operator

### ISSUE THREAD

**Issue by cnxtwang** [benmarch/spel2js#5]
Hi there, very useful library, Thank you, all!

I got one problem when i use safe navigation.
when i use multi `?.` operators like:

```
"nested?.doesNotExist?.test"
```

will throw exception: 

```
ContextDoesNotExistException: Attempting to look up property 'test' for an undefined context.
```

So is this a bug or i have some misunderstanding about safe navigation ?

**Comment by benmarch** [comment]
Hey @cnxtwang, sorry for the extremely late response to this, I must have missed it and I haven't actively worked on this project for a while.

I think I implemented safe navigation incorrectly. I'll take another pass at it and see if it fixes this issue.

**Comment by benmarch** [comment]
@cnxtwang this is fixed in 0.2.7.
